app/containers/Applications/ReviseApplicationButtonContainer.tsx

```diff
@@ -15,6 +15,42 @@ export const ReviseApplicationButton: React.FunctionComponent<Props> = ({
   relay
 }) => {
   const router = useRouter();
+  const thisVersion = Number(router.query.version);
+  const newerSubmissionExists =
+    application.latestSubmittedRevision.versionNumber > thisVersion;
+  const latestSubmissionURL = `/reporter/view-application?applicationId=${
+    encodeURIComponent(application.id)
+  }&version=${application.latestSubmittedRevision.versionNumber}`;
+  const viewLatestSubmissionButton = (
+    <>
+      <p style={{margin: '1rem 0'}}>
+        <strong>Note:</strong> There is a more recently submitted version of this application.
+      </p>
+      <a href={latestSubmissionURL}>
+        <Button>View most recent submission</Button>
+      </a>
```

Comment:
You should use a nextjs `Link` here so that the navigation is client-side

Reply:
Ah - thanks for the reminder!